LPS-199989

danielCamara02 opened the pull request:
Passed on CI in this run: https://github.com/danielCamara02/liferay-portal/pull/62

Ticket: https://liferay.atlassian.net/browse/LPS-199989

Commit:
COMMERCE-9743 Check to see if product has enabled Subscriptions before showing on store page

Commit:
COMMERCE-9742 Each subscription type should have their own line

qz1010 committed:
LPS-196539 SF rename variable name

qz1010 committed:
LPS-196539 SF rename variable name

Commit:
COMMERCE-12504 update custom values for options

ling-alan-huang committed:
LPS-196539 Rename

ling-alan-huang committed:
LPS-196539 Rename

Commit:
COMMERCE-12504 add mapper to exception when selecting invalid sku for CPDefinitionOptionValueRel

andrea-ale-sbarra committed:
COMMERCE-12504 added missing fields (yaml file)

andrea-ale-sbarra committed:
COMMERCE-12504 added missing fields

andrea-ale-sbarra committed:
COMMERCE-12504 autogenerated code (gw buildRest)

Commit:
COMMERCE-12504 use new addCPDefinitionOptionRel in ProductOptionUtil

Commit:
COMMERCE-12504 add new addCPDefinitionOptionValueRel method

Commit:
COMMERCE-12504 use new addCPDefinitionOptionValueRel in ProductOptionValueUtil

Commit:
COMMERCE-12504 gw buildService

oliv-yu committed:
LPS-200991 LPS-198243 portal-search-web: Switch date facet to only using ADTs

oliv-yu committed:
LPS-200991 LPS-198243 portal-language-lang: Update language keys for default template

oliv-yu committed:
LPS-200991 LPS-198243 portal-search-web: Add checkbox template + multiselect support for date facet

oliv-yu committed:
LPS-200991 LPS-198243 portal-search-web: Fix bug with inputDatePicker and add namespace on ADTs

oliv-yu committed:
LPS-200991 LPS-198243 portal-search-web: Fix modified facet bug with undefined datePicker for ADTs

danielCamara02 committed:
LPS-199989 Improve and rename CanPublishLocalizedContent test

danielCamara02 committed:
LPS-199989 Remove CanPublishLocalizedContent from Publications testcase

liferay-continuous-integration commented:
To conserve resources, the PR Tester does not automatically run for every pull.

If your code changes were already tested in another pull, reference that pull in this pull so the test results can be analyzed.

If your pull was never tested, comment &quot;ci:test&quot; to run the PR Tester for this pull.

danielCamara02 commented:
@suilin Since the test size has increased considerably, I put the test steps under task blocks. Hope this is ok

oliv-yu committed:
LPS-200991 LPS-198243 portal-search-web: Rename templates

suilin commented on portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/PublicationsWebContent.testcase:
```diff
@@ -139,6 +139,162 @@ definition {
+
+			WebContentNavigator.openWebContentTemplatesAdmin(siteURLKey = "Guest");
+
+			WebContentTemplates.addCP(templateName = "WC Template Name");
```
@danielCamara02, thanks for working on it, we'd better create the template with the structure that you created before, and the create order should be structure, template then web content, can you update it?

Commit:
COMMERCE-12504 add new addCPDefinitionOptionRel method

AliciaGarciaGarcia committed:
LPS-200205 create new methods to delete the saved content entries

AliciaGarciaGarcia committed:
LPS-200205 Create model listener to delete the saved content entries when deleting an user

AliciaGarciaGarcia committed:
LPS-200205 Create model listener to delete the saved content entries when deleting an asset entry

michaelhashimoto committed:
LRCI-3875 Add getLatestBuildRunEntity to BuildEntity objects

michaelhashimoto committed:
LRCI-3875 Add a create build run event within Jethr0 server

michaelhashimoto committed:
LRCI-3875 Choose a jenkins node using the SLAVE_LABEL build parameter

michaelhashimoto committed:
LRCI-3875 Sort the builds that are returned to job builds

michaelhashimoto committed:
LRCI-3875 Rename the initial build for the pull request jobs

interaminense committed:
LRAC-15023 Return null to render tooltip component correctly after upgrading recharts version

Brunno0liveira committed:
COMMERCE-9743 Add Test

noornajj committed:
LPS-201746 Separate where clause into batches when over 1000 entries for Oracle db

interaminense committed:
LRAC-15024 Add type number on AxisX after upgrading recharts version

oliv-yu committed:
LPS-200991 LPS-198243 portal-language-lang: gradlew buildLang

suilin commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

suilin commented:
ci:stop

suilin commented:
@danielCamara02, it looks good to me, seems there is a conflict, can you rebase and resend it? Thanks.

Hijky committed:
LPS-200562 Update a macro to create and publish an object with scope value

Hijky committed:
LPS-200562 Update a macro to configure export with scope value

Hijky committed:
LPS-200562 Add a testcase to support export object entries at site level

jaydawu committed:
COMMERCE-9743 Fix test

juanjofgliferay committed:
LPS-201409 - Fix. Consider permissionKey case

AliciaGarciaGarcia committed:
LPS-200205 buildService

juanjofgliferay committed:
LPS-201409 - Refactor. Simplify

AliciaGarciaGarcia committed:
LPS-201796 check if the display date (schedule date) is after the actual date and not null, if not throw new KBArticleDisplayDateException

AliciaGarciaGarcia committed:
LPS-201796 show the schedule options only if PropsValues.SCHEDULER_ENABLED is enabled

AliciaGarciaGarcia committed:
LPS-201796 buildLang

adolfopa committed:
LPS-200202 Semver

# breaking_change_report

## What modules/apps/sharing/sharing-api/src/main/java/com/liferay/sharing/display/context/util/SharingDropdownItemFactory.java
SharingDropdownItemFactory includes a new method
createCopyLinkDropdownItem(String className, long classPK,
HttpServletRequest httpServletRequest).

## Why
The Sharing menu dropdowns need to provide an additional action to
copy a link to documents.

----

brookedalton committed:
LPS-199500 Get groupId from plid if there is no groupId column

pyoo47 committed:
LRCI-3875 Don't throw Exception

qz1010 committed:
LPS-190736 Wordsmith

pei-jung committed:
LRQA-84399 Baseline

brianchandotcom committed:
LPS-200991 Wordsmith

brianchandotcom committed:
LPS-200991 Regen

brianchandotcom committed:
LPS-201796 Rename

brianchandotcom committed:
COMMERCE-12504 Wordsmith

Hijky committed:
LPS-201866 Update a macro to click the download button more stable

Songyuewen committed:
LPS-201888 Update publication status locatr

Jeremy-Ch committed:
LPS-201880 Update paths to make them flexible

Jeremy-Ch committed:
LPS-201880 Update macro to use the suitable path

Jeremy-Ch committed:
LPS-201880 Remove the unused path

Jeremy-Ch committed:
LPS-201877 Add refresh to slow down the test

danielCamara02 commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4265/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/aa4e661cf8d6e38b797edb6617bdba46e2e075e7">aa4e661cf8d6e38b797edb6617bdba46e2e075e7</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/danielCamara02/liferay-portal/tree/LPS-199989">LPS-199989</a><br/>Branch GIT ID: <a href="https://github.com/danielCamara02/liferay-portal/commit/3f5b9080e4d2e14bdfc7810f3c8a9330170244bf">3f5b9080e4d2e14bdfc7810f3c8a9330170244bf</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-18.liferay.com/job/test-portal-source-format/7264/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-source-format/builds/7264/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265//consoleFull)

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-18.liferay.com/job/test-portal-source-format/7264//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/143918
[Console](https://test-1-18.liferay.com/job/test-portal-source-format/7264//consoleFull)

liferay-continuous-integration commented:
Error has occurred while attempting to forward pull request to `brianchandotcom`. Retrying in 1 minute...
See console log for detail:[Full Console](https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-18.liferay.com/job/test-portal-source-format/7264/">test-portal-source-format#7264</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-18.liferay.com/userContent/jobs/test-portal-source-format/builds/7264/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/suilin/liferay-portal/pull/691">suilin#691</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2460215734">[master] ci:test:sf - danielCamara02 &gt; suilin - PR#691 - 2023-11-16[03:32:54]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-18.liferay.com/job/publish-testray-report/4516/">publish-testray-report#4516</a><br/></div></div>

liferay-continuous-integration commented:
Error has occurred while attempting to forward pull request to `brianchandotcom`. Retrying in 1 minute...
See console log for detail:[Full Console](https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265/">test-portal-acceptance-pullrequest(master)#4265</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-25.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/4265/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/suilin/liferay-portal/pull/691">suilin#691</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2460215754">[master] ci:test:relevant - danielCamara02 &gt; suilin - PR#691 - 2023-11-16[03:35:27]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-25.liferay.com/job/publish-testray-report/8521/">publish-testray-report#8521</a><br/></div></div>

liferay-continuous-integration commented:
Error has occurred while forwarding pull request to `brianchandotcom`.
Please try again later or contact the CI team for assistance.
See console log for details: [Full Console](https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest(master)/4265//consoleFullconsoleText)
